zeeguu/core/model/caption.py

```diff
@@ -0,0 +1,51 @@
+import sqlalchemy
+from zeeguu.core.model import db
+from zeeguu.core.model.new_text import NewText
+
+
+class Caption(db.Model):
+    __tablename__ = "caption"
+    __table_args__ = {"mysql_collate": "utf8_bin"}
+
+    id = db.Column(db.Integer, primary_key=True)
+    video = db.relationship("Video", back_populates="captions")
+    video_id = db.Column(db.Integer, db.ForeignKey("video.id"))
+
+    time_start = db.Column(db.Float)
```

Comment:
is it better to have float or int and ms? would be nice to add comment for the unit of time.